integrationtest/couchdb/src/test/java/org/hibernate/ogm/test/integration/jboss/CouchDBModuleMemberRegistrationIT.java

```diff
@@ -77,6 +77,7 @@ private static PersistenceDescriptor persistenceXml() {
 							.createProperty().name( "hibernate.ogm.datastore.database" ).value( "ogm_test_database" ).up()
 							.createProperty().name( "hibernate.ogm.datastore.create_database" ).value( "true" ).up()
 							.createProperty().name( "hibernate.search.default.directory_provider" ).value( "ram" ).up()
+							.createProperty().name( "hibernate.transaction.jta.platform" ).value( "JBossAS" ).up()
```

Comment:
Interesting, how comes that's needed now? Shouldn't Hibernate ORM figure it out by itself within WF?